enabler/src/components/payment-methods/paypal/paypal.ts

```diff
@@ -10,7 +10,7 @@ import {
 } from "../../base";
 
 export class PaypalBuilder extends PaypalBaseComponentBuilder {
-  public componentHasSubmit = true;
+  public componentHasSubmit = false;
```

Comment:
It seems to be enough to define it in `PaypalBaseComponentBuilder`. Also I cannot find where this boolean flag has been used throughout the `enabler`

Reply:
This is not used in the enabler but it is exposed to the interface to understand once the component is built if the submit() function is available.